src/components/CommentList/CommentList.jsx

```diff
@@ -1 +1,12 @@
-export const CommentList = () => <>Put the list here</>;
+import { CommentInfo } from '../CommentInfo/CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => (
+  <>
```

Comment:
```suggestion

```
No need to use fragments since you have a common parent block

Reply:
noted, thanks